eventkit_cloud/ui/views.py

```diff
@@ -126,6 +128,28 @@ def geocode(request):
     else:
         return HttpResponse(status=204, content_type="application/json")
 
+@require_http_methods(['GET'])
+def convert(request):
+    convert = Convert()
```

Comment:
Since there is not a reasonable (publicly available default) It would be helpful to return a 501 and a message saying that the setting hasn't been configured, that way if this isn't working, someone will know what to look for.